tools/ci/cibuild.sh

```diff
@@ -381,6 +381,46 @@ function rust {
   fi
 }
 
+function zig {
+  mkdir -p "${prebuilt}"/zig
+  add_path "${prebuilt}"/zig
+
+  if ! type zig &> /dev/null; then
+    case ${os} in
+      Darwin)
+        brew install zig
+        ;;
+      Linux)
+      # Currently Debian don't install zig (only built)
+        export ZIG_HOME=${prebuilt}/zig
+        cd /tmp
+	curl -Os https://ziglang.org/download/0.9.1/zig-linux-x86_64-0.9.1.tar.xz
+	tar -xf zig-linux-x86_64-0.9.1.tar.xz && rm zig-linux-x86_64-0.9.1.tar.xz && mv zig-linux-x86_64-0.9.1/* "${prebuilt}"/zig && rm -fr zig-linux-x86_64-0.9.1
+	;;
+    esac
+  fi
+}
+
+function zig {
```

Comment:
So why do we need two definitions of the same function `zig`?

Reply:
I will check! There may have been a problem during the merge. affz

Reply:
Follow my previous suggestion will simplify your life:
https://thoughtbot.com/blog/git-interactive-rebase-squash-amend-rewriting-history